src/scripts/main.js

```diff
@@ -1,3 +1,385 @@
+/* eslint-disable max-len */
 'use strict';
 
-// write code here
+const NAME = 'Name';
+const POSITION = 'Position';
+const OFFICE = 'Office';
+const AGE = 'Age';
+const SALARY = 'Salary';
+
+const DONE = 'done';
+const NOT_DONE = 'not-done';
+
+const body = document.getElementsByTagName('body')[0];
+const tbody = document.getElementsByTagName('tbody')[0];
+const headers = document.querySelectorAll('th');
+const topHeaders = Array.from(headers).slice(0, 5);
+let table = tbody.querySelectorAll('tr');
+const staff = [];
+
+let isClickedName = false;
+let isClickedPosition = false;
+let isClickedOffice = false;
+let isClickedAge = false;
+let isClickedSalary = false;
+
+table.forEach((row) => {
+  const memberStaff = {};
+
+  memberStaff.name = row.querySelectorAll('td')[0].textContent;
+  memberStaff.position = row.querySelectorAll('td')[1].textContent;
+  memberStaff.office = row.querySelectorAll('td')[2].textContent;
+  memberStaff.age = row.querySelectorAll('td')[3].textContent;
+  memberStaff.salary = row.querySelectorAll('td')[4].textContent;
+
+  staff.push(memberStaff);
+});
+
+topHeaders.forEach((header) => {
+  header.addEventListener('click', (e) => {
+    switch (e.target.textContent) {
+      case NAME:
+        const acsSortedName = staff.sort((a, b) =>
+          a.name.localeCompare(b.name),
```

Comment:
Fix these linter error to pass tests 
<img width="1190" alt="image" src="https://github.com/mate-academy/js_employees_table_DOM/assets/64750958/de161a37-03b4-49c4-b07f-5aba9271f8f2">